### DISCUSSION TITLE
Switching Pools has sent me into fulltime pending

### DISCUSSION THREAD

**Discussion by MarcusArthur** [Chia-Network/chia-blockchain#7418] [Support]
When I try and join a different pool, I get a pop-up in the Chia GUI that says -

Cannot join a pool while waiting for target state: {'owner_pubkey': '0x8fae46a6ccfee859f3ed0b7ce41595b91a573bea792cf8ebce70341013bdf7c1662c165a3258448edf5075c724a2b8a2', 'pool_url': 'https://eu1.pool.space', 'relative_lock_height': 64, 'state': 3, 'target_puzzle_hash': '0x2f2c9ba1b2315d413a92b5f034fa03282ccba1767fd9ae7b14d942b969ed5d57', 'version': 1}

Been stuck on pending for ages now, can't find any resources online to help... not sure what to do.

I've tried refreshing the GUI w/ a force-reload, didn't help.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
